aries-site/src/data/wcag/datasearch.json

```diff
@@ -0,0 +1,512 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "conditional",
+    "notes": "If an icon is provided, an aria-label can be added to the icon."
+  },
+  {
+    "rule": "1.2.1",
+    "label": "Audio-only and Video-only (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.2",
+    "label": "Captions (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.3",
+    "label": "Audio Description or Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.4",
+    "label": "Captions (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "passed",
```

Comment:
The screenreader just describes this field as 'search input' I wonder if in certain contexts if it would be beneficial for the description to give more context so that a screenreader user understands that this search input is related to the Data

Reply:
maybe we can check with Bill on this one?

Reply:
Yeah would be good to get Bill's perspective on this. 

My thought is that it would be good to allow an aria-label to be passed to search input, there are a ton of different contexts people could use Data + Friends in and it seems like some situations giving the user more specific information via aria-label about the search field would be helpful

Reply:
sounds good Ill follow up with Bill

Reply:
updated to conditional